src/sqlfluff/rules/L016.py

```diff
@@ -348,14 +366,14 @@ def generate_fixes_to_coerce(
         return fixes
 
     @staticmethod
-    def _gen_line_so_far(raw_stack, initial_buff=None):
```

Comment:
Removed the `initial_buff` parameter because there's only one call to this function, and the caller was always passing `[]`.